docs/hazmat/primitives/key-derivation-functions.rst

```diff
@@ -600,6 +600,131 @@ Different KDFs are suitable for different tasks such as:
         raises an exception if they do not match.
 
 
+.. currentmodule:: cryptography.hazmat.primitives.kdf.kbkdf
+
+.. class:: KBKDFHMAC(algorithm, mode, length, rlen, llen, location,\
+           label, context, fixed, backend)
+
+    .. versionadded:: 1.3
+
+    KBKDF (Key Based Derivation Function) is defined by NIST SP 800-108
+    in the `NIST-SP-800-108`_ document, to be used to derive additional
```

Comment:
Probably can just link the previous `NIST SP 800-108` rather than repeat it as a link.